src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1834,18 +1834,30 @@ static guint16 vector2_methods[] = {
 	SN_ctor,
```

Comment:
This method group should also be updated to cover `Quaternion` and `Plane`. I don't have enough context into Mono to know if `Plane` is problematic since it is `Vector3, float`. I imagine `Quaternion` just needs to be handled since it has an identical layout to `Vector4`. However, not all the same functions get lightup for `Quaternion` and it is non-obvious how to correctly filter things here.